packages/shared/src/lib/func.ts

```diff
@@ -214,9 +214,10 @@ export const broadcastMessage = (
   channel.close();
 };
 
-export const promisifyEventListener = <T>(
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const promisifyEventListener = <T, E = any>(
```

Comment:
Should we maybe use `unknown` instead?

Reply:
I'll have to update the others that's using `promisifyEventListener`. Will do that later 🙈